src/python/grpcio_observability/grpc_observability/_observability.py

```diff
@@ -94,12 +95,10 @@ def __init__(self, exporter: "grpc_observability.Exporter" = None):
         if exporter:
             self.exporter = exporter
         else:
-            pass
-            # Actual implementation of OC exporter
-            # open_census = importlib.import_module(
-            #     "grpc_observability._open_census")
-            # self.exporter = open_census.OpenCensusExporter(
-            #     self.config.get().labels)
+            open_census = importlib.import_module(
+                "grpc_observability._open_census_exporter")
+            self.exporter = open_census.OpenCensusExporter(
```

Comment:
We probably want to fail fast in this scenario so that people don't wonder why this is silently not exporting anything. Let's just `raise NotImplementedError()` for now.